api/apps/api/src/modules/scenarios/scenarios.controller.ts

```diff
@@ -1189,15 +1189,15 @@ export class ScenariosController {
 
   @ApiOperation({
     description: `End Scenario Editing session.`,
-    summary: `Releases the lock created on a scenario so other
+    summary: `Deletes the lock created on a scenario so other
     users can start editing the scenario.`,
   })
   @ApiNoContentResponse({
     status: 204,
-    description: 'Lock was released correctly',
+    description: 'Lock was deleted correctly',
```

Comment:
Super minor: I'd say there's no need to rephrase this - we use `DELETE` because that's the verb we get from mighty HTTP, but in domain language, we still release the lock.